includes/class-wc-stripe-webhook-handler.php

```
@@ -1316,7 +1316,14 @@ public function process_deferred_webhook( $webhook_type, $additional_data, $noti
 					}
 
 					// Check if the order is still in a valid state to process the webhook.
-					if ( ! $order->has_status( apply_filters( 'wc_stripe_allowed_payment_processing_statuses', [ OrderStatus::PENDING, OrderStatus::FAILED ], $order ) ) ) {
+					// Include cancelled and on-hold: after 3DS the customer may cancel or the order may be on-hold,
+					// but if Stripe sends a payment_intent.succeeded webhook event we should still mark the order as paid.
+					$allowed_statuses = apply_filters(
```

PHPMD (2.15.0)

[error] The variable $allowed_statuses is not named in camelCase. (undefined)

(CamelCaseVariableName)